data/routers/actions-router.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+
+const actionDb = require('../helpers/actionModel');
+
+const router = express.Router();
+
+router.get('/', validateProjectId, (req, res) => {
+  actionDb
+    .get()
+    .then(actions => {
+      res.status(200).json(actions);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({ error: 'Error retrieving actions' });
+    });
+});
+
+router.get('/:id', validateProjectId, (req, res) => {
+  actionDb
+    .get(req.action.id)
+    .then(action => {
+      res.status(200).json(action);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({ error: 'Error retreiving the action' });
+    });
+});
+
+router.post('/', validateProjectId, (req, res) => {
+  const newAction = { ...req.body, project_id: req.params.id };
```

Comment:
**From our 1:1 -**

`req.params.id` would work if we had an endpoint like `"/:id"`. In this case, our `req.params.id` would just be undefined.

For this post request, I'd instead check the `req.body` to make sure it has these three properties:
```
{
    project_id: 1,
    description: "action!",
    notes: "Extra notes here."
}
```
If the `req.body` has all the properties we need, then we can just use
```
const newAction = req.body
```
And we're good to go 💪